components/link.js

```diff
@@ -13,6 +23,14 @@ const LinkComponent = ({ children, locale, ...props }) => {
 
   let href = props.href || pathname;
 
+  if ((props.href && i18nPaths[language] && !i18nPaths[language].includes(href)) || href.includes("http", 0)) {
+    return (
+      <Link href={href} passHref>
+        {children}
+      </Link>
+    );
+  }
```

Comment:
Add comment for the usecase handled with this condtion.

Reply:
done